CHANGELOG.md

```diff
@@ -2,6 +2,18 @@
 
 ## Released
 
+## 4.3.0
+### Added
+- Ensure the repo runs only under Node 18. There were issues running it on Node 20. Also adds a change to ensure that only npm is used for managing dependencies, done by the script `npx only-allow npm`. Not a good idea to mix dependency managers on the same project, like npm and Yarn.
```

Comment:
We dont have any issues with Node 20 could you please explain what is your issue?

Reply:
I don't remember exactly which issue I had with it, but I'll test again and let you know.